### DISCUSSION TITLE
Welcome to gethouse Discussions!

### DISCUSSION THREAD

**Discussion by hussain4real** [get-house/gethouse#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JohnBahago** [comment]
Discussion, commitment and openness is key the success of this project. We can share ideas, wish and progressive solutions or views. Thank you for being part of This community
